test/browser_tests/spec_suites/about-deputy-director.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+var TheAboutDirectorPage = require( '../page_objects/page_about-director.js' );
+
+describe( 'The About Deputy Director Page', function() {
+  var page;
+
+  beforeAll( function() {
+    page = new TheAboutDirectorPage( );
+    page.get( 'deputyDirector' );
+  } );
+
+  it( 'should properly load in a browser',
+    function() {
+      expect( page.pageTitle() ).toBe( 'About Meredith Fuchs' );
+    }
+  );
+
+  it( 'should have a side nav',
+    function() {
+      expect( page.sideNav.isPresent() ).toBe( true );
+    }
+  );
+
+  it( 'should include the Deputy Director\'s bio',
```

Comment:
Trivial: Possibly nest double/single quotes here and throughout for readability:

`it( "should include the Deputy Director's bio",` or use actual apostrophe’s ’’’’’’’’’’